src/components/userProfile/CloudTag.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import {
+  StyleSheet,
+  Text,
+  View
+} from 'react-native';
+
+export default class CloudTag extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.TagCloud = this.orderData().map((item, key) => {
+      const tagContainerStyle = {
+        paddingLeft: this.getRandomPadding(item.point),
+        paddingTop: this.getRandomPadding(item.point) / 2,
+        paddingRight: this.getRandomPadding(item.point),
+        paddingBottom: this.getRandomPadding(item.point) / 2,
+      };
+
+      const tagStyle = {
+        fontSize: 12 + (item.point * 3),
+      };
+
+      return (
+        <View key={key} style={tagContainerStyle}>
+          <Text style={tagStyle}>{item.title}</Text>
+        </View>
+      );
+    });
+  }
+
+  orderData() {
+    let result = [];
+    let tagList = this.props.tagList;
+    tagList.sort((a, b) => {
+      if (a.point === b.point) return 0;
+      else return a.point > b.point ? -1 : 1;
+    });
+
+    const maxPoint = tagList[0].point;
+    let flag = true;
```

Comment:
If you leave comment about this 'flag' or rename to well explained name, will be better